python/catboostserver/README.md

```diff
@@ -0,0 +1,45 @@
+# CatBoost Server
+
+[CatBoost](https://catboost.ai/) is a high-performance open source library for gradient boosting on decision trees.
+
+To start the server locally for development needs, run the following command under this folder in your github repository.
+
+```bash
+python -m catboostserver --model_dir /path/to/model_dir --model_name catboost-model
```

Comment:
```suggestion
uv venv

uv run --active python -m catboostserver --model_dir /path/to/model_dir --model_name catboost-model
```

I think this will be more developers friendly